src/Microsoft.AspNetCore.Mvc.Core/project.json

```diff
@@ -40,6 +40,7 @@
       "type": "build",
       "version": "1.1.0-*"
     },
+    "Microsoft.Extensions.Logging": "1.1.0-preview2-22526",
```

Comment:
> Is that acceptable?

Depends. If you're using `#pragma warning disable 0618` / `#pragma warning restore 0618` around the specific lines using the old constructors and you've got at least a test or two exercising the new constructors, all is fine. It's also fine to update the test code to exclusively or almost exclusively use the new constructors.

What's not fine is suppressing that warning in `project.json`.